binding/Binding/SKCanvas.cs

```diff
@@ -952,6 +952,13 @@ public void SetMatrix (SKMatrix matrix)
 			SkiaApi.sk_canvas_set_matrix (Handle, &matrix);
 		}
 
+		public void SetMatrix (in SKMatrix matrix)
```

Comment:
Note: C# will always prefer more specific overload. Which means for 2.x builds old code will still use old `SetMatrix(SKMatrix)`, unless they specify `in` explicitly.

Reply:
One annoying thing is that this will make this method ambiguous to visual basic... VB appears not to be able to distinguish between in/ref and nothing. I know we have a scattering of ref and nothing in 2.x and I tried to clean this all up. However, maybe this is making migrations a bit more annoying.

Reply:
Unfortunately, you are right. Just tested locally, and found this issue confirming it https://github.com/dotnet/vblang/issues/505

Reply:
I guess the only choice would be to use reflection/UnsafeAccessor then. I don't really want to modify SkiaSharp 3.0 API surface.